documentation-website/Writerside/snippets/get-started-with-exposed/app/src/test/kotlin/org/example/AppTest.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * This source file was generated by the Gradle 'init' task
+ */
+package org.example
+
+import kotlin.test.Test
+import kotlin.test.assertNotNull
+
+class AppTest {
+    @Test fun appHasAGreeting() {
+//        val classUnderTest = App()
```

Comment:
Probably this test could be removed. Was it generated automaticaly with the module?

Reply:
Yes, it was auto-generated.